helm/resident/.gitignore

```diff
@@ -1,2 +1,3 @@
 charts/
-Charts.lock
+Chart.lock
+git.properties
```

Comment:
resident-services/deploy/resident
/install.sh

CHART_VERSION=1.3.0-beta.1-develop
change to 
CHART_VERSION=1.3.0-develop